zips/zip-0227.rst

```diff
@@ -369,9 +381,15 @@ For every transaction:
 
   - It MUST be the case that $0 < \mathtt{assetDescSize} \leq 512$.
   - It MUST be the case that $\mathsf{asset\_desc}$ is a string of length $\mathtt{assetDescSize}$ bytes.
-  - Elements of every Issue Note in ``IssueAction`` MUST be valid encodings of the types given in the `Issue Note`_ section, and MUST encode the same $\mathsf{AssetBase}$. 
+  - Every Issue Note in ``IssueAction`` MUST be a valid encoding of the $\mathsf{Note^{Issue}}$ type, and MUST encode the same $\mathsf{AssetBase}$. 
   - This $\mathsf{AssetBase}$ MUST satisfy the derivation from the issuance validating key and asset description described in the `Specification: Asset Identifier`_ section.
   - It MUST be the case that $\mathsf{issued\_assets}_{\mathsf{OUT}}(\mathsf{AssetBase}).\mathsf{final} \neq 1$.
+  - If $\mathsf{issued\_assets}_{\mathsf{OUT}}(\mathsf{AssetBase}).\mathsf{ref\_note} = \bot$, then let $\mathsf{note}_1$ be the first Issue Note in the Issuance Action.
```

Comment:
Note that this specification means that if the reference note already exists, we do not update it with a new reference note in case it is present in a subsequent Issuance Action. 

It seems a better procedure to not update the global state unnecessarily (a malicious Issuer could keep adding reference notes to try to DOS the network by making them constantly have to update the global issuance state)